### ISSUE TITLE
Update the examples

### ISSUE THREAD

**Issue by 0xbrayo** [ramsayleung/rspotify#479]
So many of the examples provided are broken now. Please update, there are so many breaking changes that are not addressed.

**Comment by masakk1** [comment]
I second this.
I'm trying to use the pagination_sync example and it's broken.

**Comment by MaximoMachado** [comment]
What is the error you're encountering with pagination_sync? I ran it myself locally and it worked as expected. Give this command a try: `cargo run --example pagination_sync --features="env-file cli client-ureq ureq-rustls-tls" --no-default-features`

Also can you give a list of the examples you're finding are broken?

**Comment by Jonsen94** [comment]
Just tested "auth_code".

It uses the wrong scope. "user-read-currently-playing" would be correct and works fine.

**Comment by ramsayleung** [comment]
I don't fully understand the description:

> "auth_code" uses the wrong scope, "user-read-currently-playing" would be correct and works fine.

The `auth_code` example is using exactly the `user-read-currently-playing` scope: 

https://github.com/ramsayleung/rspotify/blob/eb1260c24a74e8d3ce97cd3a6222da7b1d1989c3/examples/auth_code.rs#L36

**Comment by Jonsen94** [comment]
Yes, sorry, you are right. The code I refer to is from the comment above it (which I used for the custom callback url):

[    // Same for RSPOTIFY_REDIRECT_URI. You can also set it explictly:
    //
    // ```
    // let oauth = OAuth {
    //     redirect_uri: "http://localhost:8888/callback".to_string(),
    //     scopes: scopes!("user-read-recently-played"),
    //     ..Default::default(),
    // };
    // ```](https://github.com/ramsayleung/rspotify/blob/eb1260c24a74e8d3ce97cd3a6222da7b1d1989c3/examples/auth_code.rs#L27C1-L35C11)

Sorry for the confusion.
